src/effects/presets/effectchainpresetmanager.cpp

```diff
@@ -394,6 +394,20 @@ void EffectChainPresetManager::setPresetOrder(
                 m_effectChainPresets.value(chainPresetName));
     }
 
+    // After having changed the presets order in DlgPrefEffects, we received the
+    // new lists. The '---' was not displayed there, so it's not in the list.
+    // Make sure the empty preset "---" is the first item in the list.
```

Comment:
please use consistent quote marks in the comments just for neatness

Reply:
:+1: